flink-runtime/src/main/java/org/apache/flink/runtime/executiongraph/AccessExecutionGraph.java

```diff
@@ -158,6 +159,14 @@
 	 */
 	Map<String, SerializedValue<OptionalFailure<Object>>> getAccumulatorsSerialized();
 
+	/**
+	 * Returns a {@link BlockingPersistentResultPartitionMeta}.
+	 * @return BlockingPersistentResultPartitionMeta contains ResultPartition locations
+	 */
+	default BlockingPersistentResultPartitionMeta getBlockingPersistentResultPartitionMeta() {
```

Comment:
I think this is a tricky use of a default methods, because this is not a working implementation for all subclasses, which is how default methods on interfaces should only be used.